### PR TITLE
Bug 1618281 - Deploy What's New Badge for 75

### PR DESCRIPTION
**Pull request by piatra**
* added back the new WNP message and badge
* updated to a new body string `cfr-whatsnew-searchbar-body-topsites`
* updated all targeting to 75
* removed 1 of the older WNP messages

### CODE REVIEW

File: whats-new-panel-archived.json [hunk 1]
```diff
@@ -49,5 +49,29 @@
...
+      "body": {
+        "string_id": "cfr-whatsnew-permission-prompt-body"
+      },
+      "bucket_id": "WHATS_NEW_PERMISSION_PROMPT_72",
+      "cta_type": "OPEN_URL",
+      "cta_url": "https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/whats-new-notifications",
+      "link_text": {
+        "string_id": "cfr-whatsnew-permission-prompt-cta"
+      },
+      "published_date": 1578405980832,
```
Comment (ncloudioj) [inline whats-new-panel-archived.json]: `published_date` would be generated by Remote Setting.

File: whats-new-panel-archived.yaml [hunk 2]
```diff
@@ -37,3 +37,20 @@
...
+- id: WHATS_NEW_PERMISSION_PROMPT_72
+  content:
+    body:
+      string_id: cfr-whatsnew-permission-prompt-body
+    bucket_id: WHATS_NEW_PERMISSION_PROMPT_72
+    cta_type: OPEN_URL
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/whats-new-notifications
+    link_text:
+      string_id: cfr-whatsnew-permission-prompt-cta
+    published_date: 1578405980832
```
Comment (ncloudioj) [inline whats-new-panel-archived.yaml]: ditto.

File: whats-new-panel.yaml [hunk 3]
```diff
@@ -257,3 +240,42 @@
   template: whatsnew_panel_message
   trigger:
     id: whatsNewPanelOpened
+- id: WHATS_NEW_AWESOMEBAR_75
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_AWESOMEBAR_75
+    published_date: 1583941100700
```
Comment (ncloudioj) [inline whats-new-panel.yaml]: ditto

**Comment by piatra** [comment]
Cleared up comment that `published_date` is a WNPanel specific field that is needed.

**Pull request merged**